### PR TITLE
Add delete button in form action buttons when the request is ajax

### PR DESCRIPTION
**Pull request by tifabien**
I am targeting this branch, because it adds the delete button in form action buttons when the request is ajax.

## Changelog

```markdown
### Added
- Added delete button in form action buttons when the request is ajax
```

### CODE REVIEW

File: src/Resources/views/CRUD/base_edit_form.html.twig [hunk 1]
```diff
@@ -81,6 +81,10 @@
                         {# NEXT_MAJOR: remove default filter #}
                         {% if objectId|default(admin.id(object)) is not null %}
                             <button type="submit" class="btn btn-success" name="btn_update"><i class="fa fa-save" aria-hidden="true"></i> {{ 'btn_update'|trans({}, 'SonataAdminBundle') }}</button>
+                            {% if admin.hasRoute('delete') and admin.hasAccess('delete', object) %}
+                                {{ 'delete_or'|trans({}, 'SonataAdminBundle') }}
+                                <a class="btn btn-danger" href="{{ admin.generateObjectUrl('delete', object) }}"><i class="fa fa-minus-circle" aria-hidden="true"></i> {{ 'link_delete'|trans({}, 'SonataAdminBundle') }}</a>
```
Comment (kunicmarko20) [inline src/Resources/views/CRUD/base_edit_form.html.twig]: can you please break this line into multiple?

**Comment by tifabien** [comment]
before
after

**Comment by tifabien** [comment]
I was also surprised to not having this button in this context. I checked the delete + (confirm, cancel) steps and all is ok and stay in the modal

**Comment by greg0ire** [comment]
Thanks @tifabien !

**Pull request merged**